plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1144,6 +1153,9 @@ public String toString() {
             if (_slot  != null) {
                 netBuilder.append(String.format("<address type='pci' domain='0x0000' bus='0x00' slot='0x%02x' function='0x0'/>\n", _slot));
             }
+            if (queues > 0) {
+                netBuilder.append(String.format("<driver queues='%d'/>\n", queues));
```

Comment:
I'm seeing a lot of references adding this line with one more parameter, have you seen that?

<driver name='vhost' queues='N'/>

Reply:
I did see that. The documentation states that it is optional and if you specify a name as vhost it forces it and fails if it can't do it for whatever reason. If you leave it out, the default is still vhost, but it will fallback to qemu if it doesn't work. So I thought it would be better to leave it out and not chance angering libvirt/qemu in one of the many possible combinations of configs ACS could use. https://libvirt.org/formatdomain.html#setting-nic-driver-specific-options

Reply:
sounds good to me